index.html

```diff
@@ -0,0 +1,183 @@
+    <hr class="hr2">
+    <div class="card-section">
+        <div class="card">
+            <div class="card-info">
+            <ul class="card-content">
+                    <li><img class="img-w" src="./images/lecture.png" alt="#"></li>
+                   <li class="lpd" >Word</li>
+                 <li  class="msg">Out pouring of the spirit of God through His word by the man of God as He give utterances.</li>
+           </ul>
+        </div>
+</div>
+        
+        <div class="card">
+            <div class="card-info">
+            <ul class="card-content">
+            <li><img class="img-w" src="./images/lecture.png" alt="#"></li>
+            <li class="lpd2">Prayers</li>
+            <li><p class="msg1">Its a time to draw down the presence of God and seeking His presence through prayers.</p></li>
+        </ul>
+    </div>
+</div>
+
+        <div  class="card">
+            <div class="card-info">
+            <ul class="card-content">
+            <li><img class="img-w" src="./images/lecture.png" alt="#"></li>
+            <li class="lpd3">Worship</li>
+            <li><p class="msg2">praise and worhip secton to bring down the glory of God with annoited gospel musician.
+            </p>
+        </li>
+        </ul>
+    </div>
+</div>
+
+        <div class="card">
+            <div class="card-info">
+            <ul class="card-content">
+            <li><img class="img-w" src="./images/lecture.png" alt="#"></li>
+            <li class="lpd5">Discusion</li>
+            <li><p class="msg3">Building strong and vibrant youth for christ through moral building and responsible youth in the society</p></li>
+        </ul>
+    </div>
+</div>
+
+        <div class="card">
+            <div class="card-info">
+            <ul class="card-content">
+            <li><img class="img-w"   src="./images/lecture.png" alt="#"></li>
+            <li class="lpd4">Fasting</li>
+            <li><p class="msg4">This event will take place on the last day of worship.</p></li>
+        </ul>
+    </div>
+</div>
+    </div>
+    <div class="join">
+    <p class="us">Worship With Us</p>
+</div>
+
+<div class="join2">
+    <p><a href=""> See More Program</a></p>
+</div>
+</section>
+
+<section class="ministers">
+
+     <h6 class="speakers">Featured Speakers</h6>
+    <hr class="hr1">
+    <div  class="minister-card"  id="js-card">
+    
+ <button type="button" class="more">more<i class="arrow down"></i></button>
+    </div>
+
+</section>
+    </main>
+
+    <Footer class="footer1">
+      
+        <h2 class="fg">Partner</h2>
+         <hr class="hr-f">
+        <div class="grid"> 
+        <a href=""><img src="./images/airbnb.png" alt="#" class="footer-img "></a>
+        <a href=""><img src="./images/mozilla.png" alt="#" class="footer-img "></a>
+        <a href=""><img src="./images/naver.png" alt="#" class="footer-img "></a>
+        <a href=""><img src="./images/google.png" alt="#" class="footer-img "></a>
+        </div>  
```

Comment:
- [ ] Please add the proper spacing within the selected elements, using `grid` is easy to achieve if you use `gap` property

Your project:
![image](https://user-images.githubusercontent.com/52635378/164945140-b59e017b-5270-4025-9696-cdeaa643ce38.png)

Template:
![image](https://user-images.githubusercontent.com/52635378/164945145-cc35fee4-0e7f-4552-8475-77ffe3d106fe.png)

Reply:
done

Reply:
done